elasticdl_client/api.py

```diff
@@ -52,10 +52,16 @@ def init_zoo(args):
     tmpl_str = """\
 FROM {{ BASE_IMAGE }} as base
 
+{% if USE_LOCAL_PKG %}\
+COPY build/*.whl /
```

Comment:
It will bring the limit that the local pkg path should be under the path `./build`.
How about use `local_pkg_dir` instead of a bool switch `use_local_pkg`